### ISSUE TITLE
Just waiting to play

### ISSUE THREAD

**Issue by MoazAlkharfan** [mantou132/Spotify-Lyrics#146]
- Browser: `Edge`
- Just stuck on waiting to play.

Clicking the lyrics icon then the lyrics open and this is how it looks like without it being loaded.
Even if i supply an `.lrc` file it doesn't change from this view.

![image](https://github.com/user-attachments/assets/56c0f021-feec-4ea9-bb2d-d8e93395eea4)

**Comment by oRevived** [comment]
Make sure you aren't playing Spotify on any other devices/tabs (or using the desktop app and site simultaneously). You shouldn't see a green bar like this at the bottom.
![image](https://github.com/user-attachments/assets/fd4df9b5-2f34-4bfc-8f05-212371ef5e4a)

**Comment by nadaviko15** [comment]
You can refresh the page its work for me everytime

**Comment by MoazAlkharfan** [comment]
@oRevived You are correct, the issue is that I'm playing on my phone but want to see the lyrics on the PC because the PC has no bluetooth.

So, do you know what the problem here is?
If you can just point me in the right direction I might give it a try to solve it.

**Comment by oRevived** [comment]
Unfortunately this extension won't work if there are multiple devices playing Spotify as it is meant for you to view lyrics while also listening your PC. Maybe you could consider investing in a USB Bluetooth dongle? I've been using one for over 3 years with no issues (on Windows) and they aren't that expensive (I got a TP-link Bluetooth 5.0 dongle for £10 and it works well).

**Comment by mantou132** [comment]
[0001-Support-other-device.patch](https://github.com/user-attachments/files/21533171/0001-Support-other-device.patch)
